openapi/platform/platform.yaml

```diff
@@ -4375,6 +4452,29 @@ components:
               example: Example Automation
               description: The name of the automation
               readOnly: true
+            trigger:
+              type: string
+              description: The triggering action for the automation
+              enum:
+                - TaskDefinition-manual-api-order-trigger
+                - TaskDefinition-manual-api-account-trigger
+              example: TaskDefinition-manual-account-trigger
+            status:
+              type: string
+              description: 'The current status of the automation, one of: on, off, draining'
+              example: 'on'
+              enum:
+                - 'on'
+                - draining
+                - 'off'
+              readOnly: true
+            tags:
+              type: array
+              description: A list of user-defined tags on the automation
+              items:
+                type: string
+                example: example-tag
```

Comment:
This still seems to be broken in the preview 🤔

Reply:
i was looking for another example of `items` and saw `phoneNumbers` does it like this - the only differences i see are that the example is wrapped with `'` and it doesn't have a `readOnly` property

Reply:
Fields marked as readonly don't appear in the generated example for sending requests. It is probably a bug with Stoplight that they don't show when looking at just the model. Since we don't have a create or update operation for this resource you can remove the readonly tag.

Reply:
I submitted a report to Stoplight about the docs not showing the details of array fields.